DependencyInjection/MonologExtension.php

```diff
@@ -142,6 +142,10 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
             $definition->setConfigurator(array('Symfony\\Bundle\\MonologBundle\\MonologBundle', 'includeStacktraces'));
         }
 
+        if (null === $handler['process_psr_3_messages']) {
+            $handler['process_psr_3_messages'] = !isset($handler['handler']) && !$handler['members'];
```

Comment:
This implementation is a bit fragile

Reply:
Please suggest alternative :)

Reply:
Play with the container builder to find leaves ?
Or iirc, play with the 'is_nested' flag

Reply:
But the list of options is finished, there isn't anything more possible than what's listed in Configuration.php, isn't it?
I'm not sure adding a compiler pass to cover something that is basically static is a good idea...

Reply:
Indeed, "members" and "handler" seems to cover everything.
So basically it works. But if we add something else one day, we will have to remember to update this list. This is why I said it's a **bit** fragile.
But we can live with this ;)

Reply:
Let's live with this as you say :)